app/views/pages/homepage.html.erb

```diff
@@ -1 +1,84 @@
-<h1>Homepage</h1>
+<div class="usa-grid homepage-hero">
+  <h2>While moving is never easy, <b>move.mil</b> makes your experience just a little bit better.</h2>
+</div>
+
+<div class="usa-grid">
+  <aside class="usa-width-one-third">
+    <div class="dps-login-section">
+      <p><b>Defense Personal Property System (DPS)</b> is your portal to move scheduling and the claims process.</p>
```

Comment:
1. <b> or <strong> ?
1. What do you think about phrasing this as "The Defense Personal Property System (DPS) is where you can schedule your move and submit claims for lost or damaged household goods."